src/components/expression/expression-panel.js

```diff
@@ -107,13 +113,22 @@ genotet.ExpressionPanel.prototype.initPanel = function() {
     }.bind(this));
 
   // Input type update
-  this.container.find('#regex input').click(function() {
-    this.isRegex_ = true;
-  }.bind(this));
-  this.container.find('#string input').click(function() {
-    this.isRegex_ = false;
-  }.bind(this));
-  this.container.find('#regex input').trigger('click');
+  this.container.find('#gene-input label[name="regex"] input')
```

Comment:
Oh I see. Please ignore comments regarding input names. 

On the other hand are the double quotes redudant? I think you can do [name=regex].